### PR TITLE
Add conda-forge badge and update install instructions

### LINKED ISSUE #111: Conda-forge package ?
Hi @felix-hilden,

This is a really neat library, and I have many repositories that could benefit enormously from this functionality. I'm wondering if you would be interested in making this available via [`conda-forge`](https://conda-forge.org/). The benefit of conda-forge is it would make this package available as a compiled binary in the Anaconda ecosystem (and you'd also get another badge for your Readme!). Once it's in the conda-forge repository, upkeep is fully automated, so no extra work on your part!

If this is of interest to you, I would be willing to draft the PR and maintain the conda-forge package, with you listed as primary maintainer. Please let me know!

### PR DESCRIPTION
**Pull request by Zeitsperre**
Closes #111 

- [x] Tests written and passed
- [x] Documentation and changelog entry written, docs build passed
- [x] All `tox` checks passed

### CODE REVIEW

File: docs/src/index.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
 sphinx-codeautolink
 ===================
-|pypi| |license|
+|pypi| |conda-forge| |license|
```
Comment (felix-hilden) [inline docs/src/index.rst:3]: Let's move these under "Quick start" here and in the other two files as well

File: docs/src/release_notes.rst [hunk 2]
```diff
@@ -8,6 +8,10 @@ These release notes are based on
 sphinx-codeautolink adheres to
 `Semantic Versioning <https://semver.org>`_.
 
+0.13.0 (unreleased)
+-------------------
+- Add Conda-Forge installation instructions (:issue:`111`)
+
```
Comment (felix-hilden) [inline docs/src/release_notes.rst:14]:
```suggestion
```
Seems unnecessary, although we could tell users that we "created a Conda package" or something if you'd like!

**Review by felix-hilden — CHANGES_REQUESTED** [review]
Thanks for breaking this down! Minor things below but looking good 👍

File: readme.rst [hunk 3]
```diff
@@ -12,15 +12,21 @@ For a live demo, see our online documentation on
 sphinx-codeautolink elsewhere:
 
 - Package on `PyPI <https://pypi.org/project/sphinx-codeautolink>`_
+- Package on `Conda-Forge <https://anaconda.org/conda-forge/sphinx-codeautolink>`_
```
[the remark this answers is not in the document]
Comment (Zeitsperre) [reply readme.rst]: That's fair, haha. All good.

**Review by felix-hilden — APPROVED** [review]
Thanks for taking it all the way and quick responses!

**Pull request merged**